### DISCUSSION TITLE
What's the max kafka version we are supporting?

### DISCUSSION THREAD

**Discussion by xiaobingxia-at** [milvus-io/milvus#37380] [Q&A and General discussion]
When I use milvus operator to setup the kafka/pulsar dependency, is there any way to specify the kafka version?
And what's the max version of kafka/Pulsar that Milvus is supporting?

**Comment by xiaofan-luan** [comment]
For Pulsar we actually verified 3.0. For kafka I thought latest 3.x should be supported as well but we did't test on it

**Reply by xiaobingxia-at** [reply]
Thank you! For milvus operator yaml, is there any way to specify the kafka version?

**Reply by xiaobingxia-at** [reply]
nvm, I think I got it. What's the maximum version of Kafka we can be confident to use on production?

**Reply by xiaofan-luan** [reply]
2.x and 3.x should be supported both

**Reply by xiaobingxia-at** [reply]
Thanks! Sorry to confirm with you again :) 
Given the page: https://kafka.apache.org/downloads
The latest version is 3.8.1. But you mentioned that latest 3.x is not well tested. So I assume we can use 3.7.1?

**Comment by xiaofan-luan** [comment]
I think so. There is no reason as far as we know that can't support higher version of Kafka
